feature/experimental/qos/qos_egress_scheduler/ate_tests/egress_strict_priority_scheduler_test/README.md

```diff
@@ -0,0 +1,96 @@
+# QoS-1.3: Egress strict priority scheduler test
+
+## Summary
+
+Verify that packet drops in AF4, AF3, AF2, AF1, BE1, BE0 and NC1 according to the strict-priority test traffic table
+
+## QoS traffic test setup:
+
+*   Topology:
+
+    *   2 input interfaces and 1 output interface with the same port speed. The
+        interface can be a physical interface or LACP bundle interface with the
+        same aggregated speed.
+
+    ```
+      ATE port 1
+          |
+         DUT--------ATE port 3
+          |
+      ATE port 2
+    ```
+
+*   Counters should be also verified for each test case:
+
+    *   /qos/interfaces/interface/output/queues/queue/state/transmit-pkts
+    *   /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+    *   transmit-pkts should be equal to the number of Rx pkts on Ixia port
+    *   dropped-pkts should be equal to diff between the number of Tx and the
+        number Rx pkts on Ixia ports
+
+*   Latency:
+
+    *   Should be < 100000ns
+
+## Procedure
+
+*   Connect DUT port-1 to ATE port-1, DUT port-2 to ATE port-2 and DUT port-3 to
+    ATE port-3.
+
+*   Configuration
```

Comment:
Why is this different from the original test plan as below?

- Configure 6 forwarding classes according to the [classification table].
- Configure egress multi-level strict-priority scheduling policy for the above classes.
- Send traffic flows according to the [strict-priority test traffic table]
- Verify loss-rate for traffic on the ingress ATE interface to match the expected loss.
- Verify queue drop counters on DUT egress interface.

Reply:
I have fixed the README file